dist-material/alarm-settings.yml

```diff
@@ -50,6 +50,7 @@ rules:
     metrics-name: service_instance_resp_time
     op: ">"
     threshold: 1000
+    include-names-regex: instance\_\d+
```

Comment:
This default value may break existed versions, right?

Reply:
I write on the wrong file, I will change it into the demo file tomorrow.

Reply:
Change it into the `alarm-settings-sample.yml` file.